servicetalk-loadbalancer-experimental/src/test/java/io/servicetalk/loadbalancer/RoundRobinToDefaultLBMigrationProviderTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2024 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.loadbalancer;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Properties;
+
+import static io.servicetalk.loadbalancer.RoundRobinToDefaultLBMigrationProvider.PROPERTY_NAME;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.sameInstance;
+
+final class RoundRobinToDefaultLBMigrationProviderTest {
+
+    private final Properties properties = new Properties();
+    private final RoundRobinToDefaultLBMigrationProvider provider =
+            new RoundRobinToDefaultLBMigrationProvider(properties);
```

Comment:
In tests, you can still use `System` props, just annotate this class with `@Execution(ExecutionMode.SAME_THREAD)` and clear the property in either a finally block or in `@AfterEach` method